slides/lesson12.md

```diff
@@ -118,10 +177,43 @@ let people = [
   }
 ];
 ```
+
+---
+
+### Exercise
+
+* Step 1: Create an array with your 3 top friends (or foes)
+* Step 2: Say "hello" on console to each friend, e.g.:
+
+```txt
+hello Alice
+hello Bob
+hello Carol
+```
+
 ---
 
-### exercise
+### Exercise - cont.
 
-Create an array with 3 elements, each element should be a recipe, each recipe should have at least a name and an array of ingredients.
+* Step 1: Create an array with your 3 top friends (or foes)
+* Step 2: Say "hello" on console to each friend
+* **Step 3**: Same as Step 2, but with a `for` loop
 
 ---
+
+### Exercise
+
+Create an array containing a few numbers
+
+* Output the last element of your array
+* Output the sum of all the numbers in the array
+* Output the largest number of the array
+* Output the average (mean) of all numbers (sum of all numbers divided by amount of numbers)
+* Create a new array containing only numbers of the original array that are smaller than 5
```

Comment:
What about splitting the task number five in a new slide?  It might give them a sense of achievement after completing the first four exercises